package.json

```diff
@@ -43,6 +43,7 @@
     "test:e2e:chrome": "SELENIUM_BROWSER=chrome node test/e2e/run-all.js",
     "test:e2e:chrome:mmi": "SELENIUM_BROWSER=chrome node test/e2e/run-all.js --mmi",
     "test:e2e:chrome:flask": "SELENIUM_BROWSER=chrome node test/e2e/run-all.js --build-type flask",
+    "test:api-specs": "SELENIUM_BROWSER=chrome ts-node test/e2e/run-openrpc-api-test-coverage.ts",
```

Comment:
just wondering, does the browser make any difference in the API tests? I mean, could it be that we get different responses depending on if we are using Chrome or Firefox? I assume it should be the same, but just to make sure :thinking:

Reply:
good call out! Maybe we should have two separate flows for firefox and chrome as we do with e2e tests

Reply:
I don't believe it would make a difference. They use the same inpage provider and background RPC pipeline / engine

Reply:
True. I guess this test suite isn't actually geared at exposing the issues that might arise if say the UI fails to respond for some browser specific reason